lib/tasks/load_csv.rake

```diff
@@ -0,0 +1,128 @@
+
+desc "destroy and reload all seed data"
+task :load_csv do
+  Rake::Task["initialize"].invoke
+  Rake::Task["load_customers"].invoke
+  Rake::Task["load_invoices"].invoke
+  Rake::Task["load_merchants"].invoke
+  Rake::Task["load_items"].invoke
+  Rake::Task["load_invoice_items"].invoke
+  Rake::Task["load_transactions"].invoke
+end
+
+task initialize: :environment do
+  Rake::Task["db:drop"].invoke
+  Rake::Task["db:create"].invoke
+  Rake::Task["db:migrate"].invoke
+  require './app/models/application_record.rb'
+  require './app/models/customer.rb'
+  require './app/models/invoice.rb'
+  require './app/models/merchant.rb'
+  require './app/models/item.rb'
+  require './app/models/invoice_item.rb'
+  require './app/models/transaction.rb'
+end
+
+desc "load customers"
+task load_customers: :environment do
+  require 'csv'
+  require './app/models/application_record.rb'
+  require './app/models/customer.rb'
+  CSV.foreach('./db/data/customers.csv', :headers => true,  header_converters: :symbol, :encoding => 'UTF-8') do |row|
+    t = Customer.new
+    t.first_name = row[:first_name]
+    t.last_name = row[:last_name]
+    t.save
+    if t.save == false
+      puts "Failed to add Customer ##{row[:id]} to database"
+    end
+  end
+  puts "Customer CSV Database Upload Complete"
+end
+
+task load_invoice_items: :environment do
+  require 'csv'
+  require './app/models/application_record.rb'
+  require './app/models/invoice_item.rb'
+  CSV.foreach('./db/data/invoice_items.csv', :headers => true,  header_converters: :symbol, converters: :all, :encoding => 'UTF-8') do |row|
```

Comment:
I'm going to remove the `converters: :all`. I threw that in there while debugging but I don't think it's doing anything.